### ISSUE TITLE
Video bgfx cause Exception on Windows

### ISSUE THREAD

**Issue by katananja** [mamedev/mame#2895]
```
MAME v0.192 (mame0192-225-g7e7bff3543-dirty)
Windows 10 x64 1703 15063.726
Radeon R7 250.
```
```
Versão das configurações do Radeon - 2016.0916.1515.27418
Versão do driver - 16.40.2311-160916a-307259E
Fornecedor - Advanced Micro Devices, Inc.
Versão do driver 2D - 8.1.1.1571
Direct3D® Version - 9.14.10.01221
OpenGL® Version - 6.14.10.14008
OpenCL™ Version - Not Available
Versão do AMD Mantle - 9.1.10.0146
Versão da API do AMD Mantle - 98310
Versão do driver de áudio AMD - 10.0.1.0
Vulkan™ Driver Version - 1.3.0
Vulkan™ API Version - 1.0.21
```
Using the command below cause mame to crash, sorry for the useless stack, I can't compile a windows binary with symbols:
```
mame64 -video bgfx
Unable to load effect bgfx\effects\gui_opaque.json
Unable to load effect bgfx\effects\gui_blend.json
Unable to load effect bgfx\effects\gui_multiply.json
Unable to load effect bgfx\effects\gui_add.json
Unable to load effect bgfx\effects\screen_opaque.json
Unable to load effect bgfx\effects\screen_blend.json
Unable to load effect bgfx\effects\screen_multiply.json
Unable to load effect bgfx\effects\screen_add.json
Fatal error: BGFX: Unable to load required shaders. Please check and reinstall the bgfx folder

-----------------------------------------------------
Exception at EIP=0000000004915d3b (not found): ACCESS VIOLATION
While attempting to read memory at 0000000000000008
-----------------------------------------------------
RAX=0000000000000000 RBX=000002680aef9db0 RCX=00000000ffffffff RDX=0000000000000001
RSI=000000000c912590 RDI=0000000000000001 RBP=000000000bccdc01 RSP=000000e4059f7fe0
 R8=00000268090c3b6c  R9=000000000000001b R10=00007ffb4fe295d0 R11=000000e4059f7ec0
R12=000000e4059f82d0 R13=000000000b79fd90 R14=000000000b9b1340 R15=000000000b79fd90
Stack crawl:
  000000e4059f8000: 0000000004915d3b (not found)
  000000e4059f8030: 0000000004954d8d (not found)
  000000e4059f8060: 0000000002c025d9 (not found)
  000000e4059f80b0: 0000000002bcdea1 (not found)
  000000e4059f8100: 0000000002bba3a1 (not found)
  000000e4059f8140: 0000000004410de7 (not found)
  000000e4059f8240: 0000000004417964 (not found)
  000000e4059fede0: 0000000002c8c1ac (not found)
  000000e4059ff080: 0000000002d026b5 (not found)
  000000e4059ff210: 0000000002d02bdd (not found)
  000000e4059ff270: 0000000002c8a117 (not found)
  000000e4059ff9d0: 0000000007c579ff (not found)
  000000e4059ffaa0: 0000000000e413f8 (not found)
  000000e4059ffad0: 0000000000e4151b (not found)
  000000e4059ffb00: 00007ffb4fc02774 (BaseThreadInitThunk+0x0014)
  000000e4059ffb50: 00007ffb4fd80d51 (RtlUserThreadStart+0x0021)
```
All the required files are available, for some reason MAME doesn't find it.
```
D:\MAME> dir /b bgfx\effects\*.json
gui_add.json
gui_blend.json
gui_multiply.json
gui_opaque.json
screen_add.json
screen_blend.json
screen_multiply.json
screen_opaque.json
```

**Comment by mmicko** [comment]
Please check your ini file if it is pointing to that bgfx folder or some else. Also note that shader format have been changed so all files are updated, so please use latest shaders from bgfx folder from GIT.

**Comment by Tafoid** [comment]
In short, the bgfx folders that shipped with 0.192 are good for 0.192 only.  The files have been updated to you'll need to update your from current Git Sources.

**Comment by katananja** [comment]
Updated the bgfx folder, this fix the issue.
Thank you.
